_includes/madefriendly.html

```diff
@@ -0,0 +1,26 @@
+<!-- packages -->
+<div class="container-fluid" >
+  <div class="container pt-3 pb-3">
+    <div class="row">
+      <div class="col-lg-4 col-md-3 red-heading"><hr/></div>
+      <div class="col-lg-4 col-md-6 red-heading section-heading">
+        <h2 class="lead secondary-heading">
+          Technical computing,<br />made friendly
```

Comment:
We have explicitly removed technical computing from the REPL banner (https://github.com/JuliaLang/julia/pull/28522) seems odd to reintroduce it here.

Reply:
Agreed. I've been trying to think of something that captures a) julia is a very good choice for technical stuff b) the language is easy to use c) the community is friendly d) it's not just for technical stuff. Any suggestions? Maybe I'm trying to cram too much into one slogan.